nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/ImagingComponent.java

```diff
@@ -318,9 +315,12 @@ protected PictureView computeView(DocumentModel doc, Blob blob, PictureConversio
         String viewFilename = String.format("%s_%s.%s", title, FilenameUtils.getBaseName(blob.getFilename()),
                 extension);
         viewBlob.setFilename(viewFilename);
+        ImageInfo viewBlobImageInfo = getImageInfo(viewBlob);
         pictureViewMap.put(PictureView.FIELD_FILENAME, viewFilename);
         pictureViewMap.put(PictureView.FIELD_CONTENT, (Serializable) viewBlob);
-        pictureViewMap.put(PictureView.FIELD_INFO, getImageInfo(viewBlob));
+        pictureViewMap.put(PictureView.FIELD_INFO, viewBlobImageInfo);
+        pictureViewMap.put(PictureView.FIELD_WIDTH, viewBlobImageInfo.getWidth());
+        pictureViewMap.put(PictureView.FIELD_HEIGHT, viewBlobImageInfo.getHeight());
 
```

Comment:
as `getImageInfo` can returns a `null` value i think its better to add control on `viewBlobImageInfo ` to avoid any NPE and any break changes.